docs/static/releasenotes.asciidoc

```diff
@@ -3,13 +3,65 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-8-0-1,Logstash 8.0.1>>
 * <<logstash-8-0-0,Logstash 8.0.0>>
 * <<logstash-8-0-0-rc2,Logstash 8.0.0-rc2>>
 * <<logstash-8-0-0-rc1,Logstash 8.0.0-rc1>>
 * <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-0-1]]
+=== Logstash 8.0.1 Release Notes
+
+[[notable-8.0.1]]
+==== Notable issues fixed
+
+* Fixed long-standing monitoring incompatibility on Windows where the CPU metric was not available.
+  https://github.com/elastic/logstash/pull/13727[#13727]
+
+* Recently, users running `bin/logstash-plugin` to install or update plugins stumbled upon an issue that would prevent
+  Logstash from starting due a third-party dependency update.
+  https://github.com/elastic/logstash/issues/13777[#13777]
```

Comment:
We should also mention what was fixed: "dependency was pinned to an older version". If we don't land the conservative flag PR we can also hint that we'll be improving the way upgrades are done to prevent these situations from happening.

Reply:
This https://github.com/elastic/logstash/issues/13777 is still open. Should it be closed? 
If there's more work to be done before we can close the issue, let's call that out in release notes as _progress_ toward the issue.

Reply:
> We should also mention what was fixed: "dependency was pinned to an older version".

In the context of the plugin update/install the sinatra dependency is irrelevant (reason why I started looking towards conservative updates). For the user, it either works or not - let me see if I can fomulate smt meaningful but it seems the details are only relevant for those understanding what's going on behind the scenes here.

> This https://github.com/elastic/logstash/issues/13777 is still open. Should it be closed?

That issue in particular is resolved by the pinning - we do not have any release yet so it's a manual process.
This might be the reason why it makes sense to have it still open - easier to find for users bumping into it.
We're waiting on a Sinatra 2.2.1 release - that would mean users no longer need to be concerned.

> If there's more work to be done before we can close the issue, let's call that out in release notes as progress toward the issue.

I can try to call out a future work we plan shipping (conservative updates) but not really sure about that.

Reply:
https://github.com/elastic/logstash/pull/13802/files#r811879712